pkl/evaluator_manager_exec_unix.go

```diff
@@ -0,0 +1,34 @@
+//go:build !windows
+// +build !windows
```

Comment:
```suggestion
//go:build unix
```

Reply:
The `unix` build tag doesn't exist, these must be valid `GOOS` values. I could change this to `linux || darwin`, but I think `!windows` is actually more correct here and doesn't rule out [various other UNIX OSs](https://github.com/golang/go/blob/e39e965e0e0cce65ca977fd0da35f5bfb68dc2b8/src/internal/syslist/syslist.go#L41-L54)

Reply:
It was introduced in Go 1.19!

https://tip.golang.org/doc/go1.19#go-unix

Reply:
Huh. VSCode yelled at me when I first tried this, but I think I just needed to kick gopls for it to figure things out. Neat!